cpp/test/sg/rf_test.cu

```diff
@@ -62,7 +62,7 @@ class RfClassifierTest : public ::testing::TestWithParam<RfInputs<T>> {
       params.split_algo, params.min_samples_leaf, params.min_samples_split,
       params.min_impurity_decrease, params.bootstrap_features, params.bootstrap,
       params.n_trees, params.max_samples, 0, params.split_criterion, false,
-      params.n_streams, false, 128);
+      params.n_streams, true, 128);
```

Comment:
`RfClassifierTests` test suite is failing consistently. I have yet to figure out the root cause.

Reply:
Fixed in the latest commit